index.html

```diff
@@ -63,5 +63,211 @@ <h1>
+                <li class="card-list-element">CSS</li>
+                <li class="card-list-element">JavaScript</li>
+                <li class="card-list-element">HTML</li>
+              </ul>
+            </div>
+            <div><button class="card-button g-btn">See Project</button></div>
+          </div>
+        </div>
+        <div class="card d-flex d-flex-column d-center t-center">
+          <div class="card-photo"></div>
+          <div class="card-desc">
+            <div class="card-header">
+              <h3>Multi-Post Stories<br />Gain+Glory</h3>
+            </div>
+            <div class="card-lang">
+              <ul class="card-list d-flex d-center">
+                <li class="card-list-element">Ruby on Rails</li>
+                <li class="card-list-element">CSS</li>
+                <li class="card-list-element">JavaScript</li>
+                <li class="card-list-element">HTML</li>
+              </ul>
+            </div>
+            <div><button class="card-button g-btn">See Project</button></div>
+          </div>
+        </div>
+        <div class="card d-flex d-flex-column d-center t-center">
+          <div class="card-photo"></div>
+          <div class="card-desc">
+            <div class="card-header">
+              <h3>Multi-Post Stories<br />Gain+Glory</h3>
+            </div>
+            <div class="card-lang">
+              <ul class="card-list d-flex d-center">
+                <li class="card-list-element">Ruby on Rails</li>
+                <li class="card-list-element">CSS</li>
+                <li class="card-list-element">JavaScript</li>
+                <li class="card-list-element">HTML</li>
+              </ul>
+            </div>
+            <div><button class="card-button g-btn">See Project</button></div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section id="about-me" class="d-flex d-flex-column d-center t-center">
+        <Div class="aboutbg">
+            <h2 class="section-header">About me</h2>
+            <p>
+              I can help you build a product ,<br class="bnn">
+              feature or website Look through<br class="bnn">
+              some of my work and experience! If<br class="bnn">
+              you like what you see and have a<br class="bnn">
+              project you need coded,<br class="bnn">
+              don’t hestiate to contact me.
+            </p>
+            <button class="card-button btn-resume g-btn">Get My Resume</button></Div>
+    </section>
+    <section id="feature" class="t-center">
+      <div class="lang-card d-flex d-flex-column d-center">
+        <div id="lang-card-icon" class="feature-icon"></div>
+        <h3 class="feature-header">Languages</h3>
+        <div id="lang-id" class="lang-card-specs">
+          <ul class="d-flex t-center">
+            <li>JavaScript</li>
+            <li>Ruby</li>
+            <li>HTML</li>
+            <li>CSS</li>
+          </ul>
+        </div>
+      </div>
```

Comment:
- [ ] Kindly, change the main `div` tag to a `ul` tag, in this case, its child should be changed to `li`, but remember, the technologies have to stay in a ul container and li for each technology, so do not change that section (line 198-203), do the same for all cards in the about me section. > Ref:`id: "feature"`

Reply:
👍🏽 All good now.